test/16_recode.jl

```diff
@@ -173,6 +187,20 @@ end
     end
 end
 
+@testset "Group recoding array with missings, default and with missing as a key pair from $(typeof(x)) to $(typeof(y))" for
```

Comment:
Mention in the test name that `missing` is the first entry in a collection LHS.

Reply:
fixed